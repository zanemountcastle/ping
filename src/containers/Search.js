import React, { Component } from 'react';
import { StyleSheet, Text, View, ScrollView, Alert, ListView, Platform  } from 'react-native';
import { Header } from 'react-native-elements';

export default class Search extends Component {

  render() {
      return (
        <View style={styles.container}>
          <Header
            centerComponent={{ text: 'Search', style: { color: '#4D4C4C', fontSize: 20, fontWeight: '700' } }}
            outerContainerStyles={{ backgroundColor: '#fff' }}
          />
          <View style={styles.body}>
            <Text>Search page!</Text>
          </View>
        </View>
      );
    }
}

const styles = StyleSheet.create({
  container: {
    flex: 1,
    backgroundColor: '#fff',
    flexDirection: 'column',
  },
  body: {
    flex: 1,
    flexDirection: 'column',
    alignItems: 'center',
    justifyContent: 'center',
  },
});
