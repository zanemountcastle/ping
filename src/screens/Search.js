import React, { Component } from 'react';
import { StyleSheet, Text, View } from 'react-native';

export default class Search extends Component {

  render() {
    return (
      <View style={styles.container}>
        <Text>Search page!</Text>
      </View>
    );
  }
}

const styles = StyleSheet.create({
  container: {
    flex: 1,
    backgroundColor: '#fff',
    alignItems: 'center',
    justifyContent: 'center',
  },
});
