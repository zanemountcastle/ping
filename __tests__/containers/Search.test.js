import React from 'react';
import Search from '../../src/containers/Search';

import ShallowRenderer from 'react-test-renderer/shallow';

describe('search screen', () => {

  it('renders without crashing', () => {
    // Shallow render <App /> and not it's child components
    // Standard react-test-renderer doesn't allow shallow rendering
    const renderer = new ShallowRenderer();
    expect(renderer.render(<Search />)).toBeTruthy();
  });

  it('renders correctly', () => {
    const renderer = new ShallowRenderer();
    expect(renderer.render(<Search />)).toMatchSnapshot();
  });

});
