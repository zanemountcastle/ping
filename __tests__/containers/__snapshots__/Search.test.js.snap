// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`search screen renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#fff",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    disabled={false}
    ellipsizeMode="tail"
  >
    Search page!
  </Text>
</View>
`;
